tb_plugin/fe/src/components/charts/TableChart.tsx

```diff
@@ -69,7 +69,11 @@ export const TableChart: React.FC<IProps> = (props) => {
 
     const chart = new google.visualization.Table(element)
 
+    if (element.offsetHeight > 0) {
```

Comment:
it seems a little bit hack to work around this issue by simply set the height before redraw and reset after redraw?

Are there any better approach?

Reply:
Yes it's definitely a bit of a hack but I think it might be the easiest solution that doesn't require editing the Google Charts library.

Reply:
could we add some comments about the reason why use the hack way?